repos/system_upgrade/el7toel8/actors/checknis/actor.py

```diff
@@ -0,0 +1,121 @@
+from leapp.actors import Actor
+from leapp.reporting import Report
+from leapp.libraries.common.reporting import report_generic
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.common.reporting import report_with_remediation
+import ipaddress
+import re
+
+def get_line(it):
+    line = next(it)
+
+    if line.endswith("\\\n"):
+        line = line[:-2] + " " + get_line(it)
+
+    return line
+
+class CheckNis(Actor):
+    """
+    Check configuration of ypbind and nsswitch. Address of NIS server cannot be
```

Comment:
Summary line and rest of the docstring should be separated by a blank line, see [Contribution guidelines](https://github.com/oamg/leapp-repository/blob/master/CONTRIBUTING.md#code-style) and [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).